### PR TITLE
Claudia 6148 script for fiware things

### PR DESCRIPTION
**Pull request by hmunfru**
REVIEWERS

@flopezag @jesuspg 

DESCRIPTION

Creation of a script for uploading murano specific requirements

TESTING

Locally

### CODE REVIEW

**Comment by jesuspg** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -57,6 +57,26 @@ It involves mainly the addition of:
...
+#### Installing FIWARE Murano requirements by an script
+fiware-murano repository contains a script which installs all the FIWARE specific requirements. This script is in charge of:
+- creating the virtualenv
+- installing python dependences,
+- generating the core library zip file
+- uploading into murano instance
+
+To execute it, it is required to export admin credentials to access to the Cloud, where Murano is part of.
+
+    $ export OS_USERNAME='admin'
```
Comment (flopezag) [inline README.md]: I do not write 'admin' for user and tenant name and live only the corresponding description like the other.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,26 @@ It involves mainly the addition of:
 -   the murano agent configuration file: It includes different murano-agent configuration files for services to be included in the different
   linux distributions.
 
+#### Installing FIWARE Murano requirements by an script
+fiware-murano repository contains a script which installs all the FIWARE specific requirements. This script is in charge of:
+- creating the virtualenv
+- installing python dependences,
+- generating the core library zip file
+- uploading into murano instance
+
```
Comment (flopezag) [inline README.md:66]: - creating the virtualenv.
- installing python dependences.
- generating the core library zip file.
- uploading into murano instance.

Nevertheless, appart from the creation of the virtualenv, the oder do not have a clear step what the user need to do, I suggest to put the concrete command to execute them, specially in step 2 or 3 where it is the first time, I suppose, that we talk about core library and murano instance.

File: README.md [hunk 3]
```diff
@@ -73,7 +93,7 @@ We create a zip file
     $ zip -r ../../io.murano.zip *
     $ cd ./../../
 
-Finally, we execute the commando package-import for the murano client library (cosidering we are configuring murano agains FIWARE Lab)
+Finally, we execute the command package-import for the Murano client library (considering we are configuring Murano against FIWARE Lab)
 
     $ tox -e venv -- murano --murano-url http://localhost:8082 --os-username admin --os-password $PASSWORD \
```
Comment (flopezag) [inline README.md]: --os-username $OS_USERNAME
--os-tenant-name $OS_TENANT_NAME
--os-auth-url $OS_AUTH_URL

File: doc/admin.md [hunk 4]
```diff
@@ -14,17 +14,26 @@ If you planned to install Murano from the OpenStack you can follow the oficial
...
-
--   the FiwareMuranoInstance instance: this is a Murano instance which includes the region and the GE NID (in case required) in the deployed VMs. This is
- required for FIWARE monitoring.
--   the script key: There is a support script which includes a support key to be used by region administrators to give support to users.
--   the murano agent configuration file: It includes different murano-agent configuration files for services to be included in the different
-  linux distributions.
-
+### Installing FIWARE Murano requirements by an script
+fiware-murano repository contains a script which installs all the FIWARE specific requirements. This script is in charge of:
+- creating the virtualenv
```
Comment (flopezag) [inline doc/admin.md]: Idem that README file

File: README.md [hunk 5]
```diff
@@ -73,7 +93,7 @@ We create a zip file
     $ zip -r ../../io.murano.zip *
     $ cd ./../../
 
-Finally, we execute the commando package-import for the murano client library (cosidering we are configuring murano agains FIWARE Lab)
+Finally, we execute the command package-import for the Murano client library (considering we are configuring Murano against FIWARE Lab)
```
Comment (flopezag) [inline README.md]: No no, it is not finally. Finally is the execution of the test that you make after command package import. Nevertheless, user do not see the command... (missing command).

File: doc/admin.md [hunk 6]
```diff
@@ -14,17 +14,26 @@ If you planned to install Murano from the OpenStack you can follow the oficial
...
+
+    $ export OS_USERNAME='admin'
+    $ export OS_PASSWORD={the password for admin user}
+    $ export OS_TENANT_NAME="admin"
+    $ export OS_REGION_NAME={the region}
+    $ export OS_AUTH_URL={the auth url for keystone}
+
+Then, just go to folder scripts and execute the script:
+    $ cd scripts
+    $ ./upload_fiware_things.sh
```
Comment (flopezag) [inline doc/admin.md:34]: This file have to be executed always inside scripts directory. If you keep the script as is, I suggest to add a Note explaining it.

**Comment by flopezag** [comment]
LGTM

**Pull request merged**